assets/js/src/tooltip.js

```diff
@@ -1,4 +1,25 @@
-const cmds = document.querySelectorAll(".language-dockerfile span.k");
+const keywords = [
+  "ADD",
+  "ARG",
+  "CMD",
+  "COPY",
+  "ENTRYPOINT",
+  "ENV",
+  "EXPOSE",
+  "FROM",
+  "HEALTHCHECK",
+  "LABEL",
+  // "MAINTAINER",
```

Comment:
Left it out on purpose since it's deprecated and the heading ID doesn't translate 1:1 with the instruction name.